kml.cc

```diff
@@ -1506,13 +1506,28 @@ void KmlFormat::kml_mt_simple_array(const route_head* header,
     case wp_field::igc_gfo:
     case wp_field::igc_siu:
     case wp_field::igc_acz: {
-      double value = fs_igc->get_value(member).value();
-      if (global_opts.debug_level >= 6) {
-        printf(MYNAME ": Writing KML SimpleArray data: %s of %f\n", name, value);
+      if (fs_igc) {
+        if (fs_igc->has_value(member)) {
```

Comment:
Instead of implementing has_value(KmlFormat::wp_field type), or a QHash/QMap (which will fail with Qt5 as we discovered previously),
if (fs_igc && fs_igc->get_value(member).has_value()) {
  double value = fs_igc->get_value(member).value();
  ...
} else {
...
}